rebasehelper/helpers/download_helper.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import os
+import sys
+import time
+import urllib
+
+import requests
+import six
+
+from six.moves import urllib
+
+from rebasehelper.exceptions import DownloadError
+from rebasehelper.logger import logger
+
+
+class DownloadHelper(object):
+
+    """Class for downloading files and getting Response objects."""
+
+    @staticmethod
+    def progress(download_total, downloaded, start_time):
+        """Prints download progress and expected remaining time of the download directly to the standard output.
+
+        Args:
+            download_total (int, float): Size of the file which is being downloaded.
+            downloaded (int, float): Size of the part of the already downloaded part of the file.
+            start_time (float): Time when the download started.
```

Comment:
It should be mentioned in what units this is supposed to be - it's `seconds since epoch` (UNIX timestamp).